Source/WebCore/css/CSSFontSelector.h

```diff
@@ -102,6 +104,7 @@ class CSSFontSelector final : public FontSelector, public CSSFontFace::Client, p
     std::optional<AtomString> resolveGenericFamily(const FontDescription&, const AtomString& family);
 
     const FontPaletteValues& lookupFontPaletteValues(const AtomString& familyName, const FontDescription&);
+    Ref<FontFeatureValues> lookupFontFeatureValues(const AtomString& familyName);
```

Comment:
The word "lookup" is a noun, but this and the function above use it as a verb. That should be two words, "look up", so written `lookUp` instead. I also don’t understand why we need to call it "look up"; usually we just call this "find" or don’t use a verb at all.

Reply:
I agree, but I think that there is some value in keeping the wording already shipped. Having both `lookup` and `lookUp` functions right next to each other feels kinda wrong. Maybe we should change them both in a followup patch ?